dagrunner/execute_graph.py

```diff
@@ -124,25 +83,37 @@ def plugin_executor(
     Raises:
     - ValueError: If the `call` argument is not provided.
     """  # noqa: E501
-    logger.client_attach_socket_handler()
+    if CONFIG["dagrunner_logging"].pop("enabled", False) is True:
+        logger.client_attach_socket_handler(CONFIG["dagrunner_logging"])
 
     if common_kwargs is None:
         common_kwargs = {}
     common_kwargs.update({"verbose": verbose, "dry_run": dry_run})
 
-    args = [
-        arg for arg in args if arg is not None
-    ]  # support plugins that have no return value
+    # support plugins that have no return value
+    args = list(filter(lambda x: x is not None, args))
     if call is None:
         raise ValueError(
             f"call is a required argument\nnode_properties: {node_properties}"
```

Comment:
Is this a typo? 'argument/nnode_properties' should be 'argument/node_properties ?

Reply:
No it's a newline :) 👍